ci-operator/config/openshift/cluster-svcat-controller-manager-operator/openshift-cluster-svcat-controller-manager-operator-master.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+
+canonical_go_repository: github.com/openshift/cluster-svcat-controller-manager-operator
+
+binary_build_commands: make build
+test_binary_build_commands: make build GOFLAGS='-race'
+
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-svcat-controller-manager-operator
+
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+
+tests:
+- as: verify
+  commands: |
+    go get -u golang.org/x/tools/imports
+    make verify -k
+  container:
+    from: bin
+
+- as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    go get -u github.com/openshift/release/tools/gotest2junit
+    make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml
+  artifact_dir: /tmp/artifacts
+  container:
+    from: test-bin
+
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+
+- as: e2e-aws-operator
+  commands: make test-e2e
+  openshift_installer_src:
+    cluster_profile: aws
+
+resources:
+  '*':
+    limits:
+      cpu: '6'
```

Comment:
I know I approved the previous one which had the same but CI jobs should not have CPU limits now unless it's 100% intentional. See https://github.com/openshift/release/pull/2821#issuecomment-462393233